Update gui.py

Jlance999 opened the pull request:
Added frame for easier GUI manipulation in the future. Changed background color and formatted label outputs for readability.

Jlance999 committed:
Update gui.py

Added frame for easier GUI manipulation. Changed background color and formated label outputs for readability.

Jlance999 committed:
Added Logo, Header

Added logo and header labels. Also re worded/cleaned up other labels. Used PIL to resize logo image for use within GUI, this is still easily changeable.

deathbeam commented:
I personally dont think adding logo is needed, its just unnecessary clutter, the interface should be unintrusive and compact

Jlance999 committed:
Update gui.py

Removed logo image from header.

Added header and footer background color labels

Rows of data now alternate color for increased readability, thanks @NaiveHickage for the sugguestion in discord.

Commented out use of currency images until a future iteration as it was causing issues.

Jlance999 commented:
> I personally dont think adding logo is needed, its just unnecessary clutter, the interface should be unintrusive and compact

Removed in subsequent commit. The main reason for filling that space was not having the cursor cover up some of the text presented. I added a single line space between the cursor and where data begins, and may use a more transparent, watermark-esk logo within that space so it just doesn't look so empty.
